index.js

```diff
@@ -257,12 +252,13 @@ app.action(/.*?/, async (args) => {
         where: { email },
         orderBy: { createdAt: 'desc' },
       })
-      if (invite?.welcome_message == "I'm going to Epoch!") {
+      if (invite?.event) {
+        const ref = invite?.event
         await prisma.user.update({
           where: { user_id: user },
           data: { club_leader: false },
         })
-        await addToChannels(user, true)
+        await addToChannels(user, true, ref)
```

Comment:
We should simplify this and pass one string param like:

await addToChannels(user, event) // event = invite?.event

The code above should still work as long as ref is replaced with event

Reply:
That makes more sense; I'm not sure why I made another param